plugins/discourse-presence/assets/javascripts/discourse/components/topic-presence-display.js.es6

```diff
@@ -48,6 +48,15 @@ export default Ember.Component.extend({
 
   @computed("presenceUsers", "currentUser.id")
   users(users, currentUserId) {
+    const currentUser = this.get("currentUser");
```

Comment:
Feel like this could be done like this:

```
@computed("presenceUsers", "currentUser.{id,ignored_users}")
users(users, currentUser) {
    const ignoredUsers = currentUser.ignored_users;
    if (
      ignoredUsers &&
      ignoredUsers.length > 0 &&
      ignoredUsers.includes(currentUser.username)
    ) {
      return [];
    }

    return (users || []).filter(user => user.id !== currentUser.id);
```

Reply:
```
let ignored_users = null;
console.log(ignored_users || []); // []

ignored_users = undefined;
console.log(ignored_users || []); // []

ignored_users = ["bob"];
console.log(ignored_users || []); // ["bob"]
```

Reply:
Actually, my logic isn't correct, I will fix it :)